src/api.raml

```diff
@@ -1270,6 +1270,7 @@ traits:
               - article
               - blog-article
               - collection
+              - digests
```

Comment:
Hang on, needs to be singular!

Reply:
I had completely overlooked that. I'll double the corresponding changes in my elife-metrics PR